### PR TITLE
mui: change style numberfield, filter form & header infinite list

### PR DESCRIPTION
**Pull request by killui**

### CODE REVIEW

File: src/mui/list/FilterForm.js [hunk 1]
```diff
@@ -14,15 +14,14 @@ import translate from '../../i18n/translate';
...
         display: 'flex',
-        justifyContent: 'flex-end',
-        alignItems: 'flex-end',
+        justifyContent: 'flex-start',
+        alignItems: 'flex-start',
         flexWrap: 'wrap',
     },
     body: { display: 'flex', alignItems: 'flex-end' },
-    spacer: { width: 48 },
+    spacer: { width: 48, display: 'none' },
```
Comment (Claire-Cleany) [inline src/mui/list/FilterForm.js]: Is the width useful with display none?

File: src/mui/list/InfiniteList.js [hunk 2]
```diff
@@ -108,9 +108,25 @@ export class InfiniteList extends List {
         );
         return (
             <Paper style={{ background: 'transparent' }} zDepth={0}>
-                <div style={styles.header}>
-                    <ViewTitle title={titleElement} style={styles.title} />
+                <div>
```
Comment (thomas-cleany) [inline src/mui/list/InfiniteList.js:111]: That div contains all the header. Should she take the prop style.header ?

Comment (killui) [reply src/mui/list/InfiniteList.js:111]: No, or maybe I should rename the style

Comment (thomas-cleany) [reply src/mui/list/InfiniteList.js:111]: I guess so, it's a little bit confusing

File: src/mui/list/FilterForm.js (hunk 1 above)
Comment (killui) [reply src/mui/list/FilterForm.js]: not really, corrected

**Review by Claire-Cleany — APPROVED** [review]

File: src/mui/detail/Show.js [hunk 3]
```diff
@@ -79,7 +79,7 @@ export class Show extends Component {
                             href={`#/${this.props.resource}`}
```
Comment (thomas-cleany) [inline src/mui/detail/Show.js]: for the "bug" on the log resource you may want to be able to pass a custom redirection link

**Review by thomas-cleany — APPROVED** [review]

**Review by Claire-Cleany — APPROVED** [review]

**Pull request merged**